noir-projects/noir-protocol-circuits/crates/public-kernel-lib/src/public_kernel_teardown.nr

```diff
@@ -548,23 +548,32 @@ mod tests {
     }
 
     #[test]
-    unconstrained fn correctly_updates_revert_code() {
+    unconstrained fn correctly_updates_revert_code_0() {
```

Comment:
I needed to split this into multiple tests because I was getting a panic on "stack too deep".